source/options.tsx

```diff
@@ -114,9 +114,12 @@ async function validateToken(): Promise<void> {
 	reportStatus({text: 'Validating…', tokenType});
 
 	try {
+		const scopes = await getTokenScopes(tokenField.value);
+		const text = `👤 @${await getNameFromToken(tokenField.value)}`;
```

Comment:
Better parallelize them:
```suggestion
		const [scopes, user] = await Promise.all([
			getTokenScopes(tokenField.value),
			getNameFromToken(tokenField.value),
		]);
```

The emoji is part of the UI so it should be added near the DOM/HTML generation

Reply:
> The emoji is part of the UI so it should be added near the DOM/HTML generation

I'm sorry, I'm unsure how to do that, I can't find any place where the HTML is generated. The images seem to get added through CSS